proto/v1/vulnerability.proto

```diff
@@ -120,6 +120,11 @@ message Details {
 
   // Output only. URLs related to this vulnerability.
   repeated grafeas.v1.RelatedUrl related_urls = 7;
+
+  // The distro assigned severity for this vulnerability when that is
+  // available and note provider assigned severity when distro has not yet
+  // assigned a severity for this vulnerability.
+  Severity effective_severity = 8;
```

Comment:
@aysylu any thoughts on the name of this field?

Reply:
I'm fine with `effective` in the name, or `distro_severity`. Is it important for us to distinguish whether distro has already assigned severity for a vulnerability, or whether we're using a note provider's value?

Reply:
I would prefer not to call it distro_severity, since it will be populated with the severity from cvss scores when distro severity is unavailable. I believe it is ok for us to not distinguish which source of the severity we are using to populate the field.